scripts/tasks.sh

```diff
@@ -272,15 +304,6 @@ function join_token() {
     # get the kurl url and installer id from the kurl-config configmap
     local kurl_url=$(kubectl -n kube-system get cm kurl-config -ojsonpath='{ .data.kurl_url }')
     local installer_id=$(kubectl -n kube-system get cm kurl-config -ojsonpath='{ .data.installer_id }')
-    local service_cidr=$(kubectl -n kube-system get cm kurl-config -ojsonpath='{ .data.service_cidr }')
-    local pod_cidr=$(kubectl -n kube-system get cm kurl-config -ojsonpath='{ .data.pod_cidry }')
```

Comment:
this has an extra `y` `.data.pod_cidry`

Reply:
That would create problems on proxy installs with non-default pod-cidrs